examples/translation-chatbot/bot.py

```diff
@@ -166,21 +164,32 @@ async def main():
         async def on_transcript_update(processor, frame):
             await transcript_handler.on_transcript_update(processor, frame)
 
+        rtvi = RTVIBotTranscriptionProcessor()
```

Comment:
This should be 

```
rtvi = RTVIProcessor()
```

the other processors are deprecated now and this is the processor expected by the `RTVIObserver`